Taxi/Taxi/Presenter/AddTaxiParty/AddTaxiParty.swift

```diff
@@ -42,7 +42,252 @@ struct AddTaxiParty: View {
+                toNextStep()
+            } label: {
+                Text("\(String(format: "%02d", startHour)):\(String(format: "%02d", minute))")
+                    .info()
+                    .padding(EdgeInsets(top: 10, leading: 26, bottom: 10, trailing: 26))
+                    .roundedBackground(startMinute == minute)
+            }
+        } else {
+            Text("00:\(String(format: "%02d", minute))")
+                .foregroundColor(.customGray)
+                .padding(EdgeInsets(top: 10, leading: 26, bottom: 10, trailing: 26))
+                .background(RoundedRectangle(cornerRadius: 10).fill(Color.lightGray))
+        }
+    }
+}
+// MARK: - 출발 장소 선택
+extension AddTaxiParty {
+    private var departurePicker: some View {
+        HStack {
+            if let destination = destination {
+                ForEach(Place.departures(of: destination), id: \.self) { place in
+                    Button {
+                        departure = place
+                        toNextStep()
+                    } label: {
+                        Text(place.rawValue)
+                            .info()
+                            .frame(maxWidth: .infinity)
+                            .padding(.vertical, 14)
+                            .roundedBackground(departure == place)
+                    }
+                }
+            }
+        }
+        .frame(maxWidth: .infinity)
+        .padding(EdgeInsets(top: 24, leading: 18, bottom: 24, trailing: 18))
+        .background(Color.addBackground)
+        .toInfoContainer(title: "어디서 모이시나요?", selectedInfo: departure?.rawValue ?? "", toggle: step == .departure) {
+            changeStep(to: .departure)
+        }
+    }
+}
+// MARK: - 정원 선택
+extension AddTaxiParty {
+    private var personNumberPicker: some View {
+        HStack {
+            ForEach(2..<5, id: \.self) { number in
+                Button {
+                    maxNumber = number
+                    toNextStep()
+                } label: {
+                    Text("\(number)인")
+                        .info()
+                        .frame(maxWidth: .infinity)
+                        .padding(.vertical, 14)
+                        .roundedBackground(maxNumber == number)
+                }
+            }
+        }
+        .frame(maxWidth: .infinity)
+        .padding(EdgeInsets(top: 24, leading: 18, bottom: 24, trailing: 18))
+        .background(Color.addBackground)
+        .toInfoContainer(title: "최대 몇명을 모으고 싶으신가요?", selectedInfo: maxNumberToString(), toggle: step == .personNumber) {
+            changeStep(to: .personNumber)
+        }
+    }
+
+    private func maxNumberToString() -> String {
+        if let maxNumber = maxNumber {
+            return "\(maxNumber)인"
+        } else {
+            return ""
+        }
+    }
+}
+// MARK: - 선택, 비선택 RoundedRectangle 을 반환하는 함수입니다.
+extension AddTaxiParty {
+    struct RoundedBackground: ViewModifier {
+        private let selected: Bool
```

Comment:
```suggestion
        private let isSelected: Bool
```

Bool 스러운 명명이 좋을것 같슴댜 !

Reply:
확인했습니다!

Reply:
8b863738e518ed261bb4f20a1dc3a61c62879a2f 에 반영했습니다!